src/View/Helper/FormHelper.php

```diff
@@ -286,19 +292,34 @@ public function __construct(View $View, array $config = [])
      * @param \Cake\View\Widget\WidgetRegistry|null $instance The registry instance to set.
      * @param array $widgets An array of widgets
      * @return \Cake\View\Widget\WidgetRegistry
+     * @deprecated 3.6.0 Use FormHelper::widgetLocator() instead.
      */
     public function widgetRegistry(WidgetRegistry $instance = null, $widgets = [])
+    {
+        trigger_error('widgetRegistry is deprecated, use widgetLocator instead.', E_USER_DEPRECATED);
```

Comment:
We have deprecationsWarnings() helper for these cases

Reply:
Didn't know if I had to use that, I thought it was for 4.0
I based on this deprecation warning
https://github.com/cakephp/cakephp/blob/8ca5054ad237650dec85bc69377cb64a95f59f8f/src/View/Helper/FormHelper.php#L383-L385